src/styles/sass/base/_base.scss

```
@@ -0,0 +1,13 @@
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-family: $font-family;
+}
+
+// CS-A1-01: For testing purposes only. It will be removed later.
```

```suggestion
// TODO: CS-A1-01: For testing purposes only. It will be removed later.
```
Add `TODO:` as well as shown above. There can be several types of comments, like `NOTE` or `FIXME`. You can explore further on comment types.